src/org/thoughtcrime/securesms/ConversationListFragment.java

```diff
@@ -153,21 +149,28 @@ public void onActivityCreated(Bundle bundle) {
 
     setHasOptionsMenu(true);
     initializeFabClickListener(false);
-    initializeListAdapter();
+    list.setAdapter(new ConversationListAdapter(getActivity(), GlideApp.with(this), locale, this));
+    loadChatlist();
+    chatlistJustLoaded = true;
   }
 
   @Override
   public void onResume() {
     super.onResume();
 
     updateReminders();
-    list.getAdapter().notifyDataSetChanged();
+
+    if (getActivity().getIntent().getIntExtra(RELOAD_LIST, 0) == 1
+        && !chatlistJustLoaded) {
+      loadChatlist();
+    }
+    chatlistJustLoaded = false;
 
     reloadTimer = new Timer();
     reloadTimer.scheduleAtFixedRate(new TimerTask() {
       @Override
       public void run() {
-        Util.runOnMain(() -> { list.getAdapter().notifyDataSetChanged(); });
+        Util.runOnMain(() -> loadChatlist());
```

Comment:
this change is not correct, i think.

before, just the cells were redrawn, to update relative timestamps - with the change, the data are actually reloaded, which is not needed.

(coming over that when targeting https://github.com/deltachat/deltachat-android/issues/1994)